.github/workflows/desktop.yml

```diff
@@ -17,30 +17,31 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [windows-latest, ubuntu-latest, macos-latest]
+        os: [ubuntu-latest, macos-latest]
```

Comment:
Removing windows-latest from here makes the workflow slightly less readable as we have to pay attention to the special include cases.

Reply:
Either special include cases, or special exclude cases. This seemed like the lesser of two evils: we'd have to include 3 exclude statements for windows, each of which hits two permutations. This way we also get to remove two additional exclude statements for dynamic msvc runtime with macos and linux. 

I was having difficulties figuring out which jobs this config queued, or how many.

Reply:
I added some brief comments to the include/exclude statements to make it easier their purposes, especially to disambiguate between includes for the sake of adding jobs versus includes for the sake of adding variables to existing jobs.

Reply:
Perfect. That should help. Thanks!